src/sentry/static/sentry/app/views/settings/incidentRules/triggers/chart/thresholdsChart.tsx

```diff
@@ -189,7 +190,7 @@ export default class ThresholdsChart extends React.PureComponent<Props, State> {
         invisible: position === null,
         draggable: false,
         position: [yAxisSize, position],
-        shape: {y1: 1, y2: 1, x1: 0, x2: graphAreaWidth},
+        shape: {y1: 1, y2: 1, x1: 7, x2: graphAreaWidth},
```

Comment:
If this `7` is same as the other `7`s below putting this in a descriptive const would be better here, like `graphAreaMargin` or similar